src/share/poudriere/image.sh

```diff
@@ -44,7 +45,7 @@ poudriere image [parameters] [options]
 Parameters:
     -j jail         -- Jail
     -t type         -- Type of image can be one of
-                    -- iso, iso+mfs, iso+zmfs, usb, usb+mfs, usb+zmfs,
+                    -- hybrid, iso, iso+mfs, iso+zmfs, usb, usb+mfs, usb+zmfs,
```

Comment:
I think we need to call this `hybridiso` to avoid confusion, because hybrid could mean a mix of anything and is probably not specific enough for this context.